demos/continuous_batching/README.md

```diff
@@ -309,8 +309,8 @@ Benchmark duration (s):                  447.62
 Total input tokens:                      215201
 Total generated tokens:                  198588
 Request throughput (req/s):              2.23
-Input token throughput (tok/s):          480.76
```

Comment:
Can we hardcode vllm version or shouldn't we? @dtrawins @porlows1

Reply:
I can change git clone in this demo to use release tag 0.6.0 https://github.com/vllm-project/vllm/releases/tag/v0.6.0

Reply:
> Can we hardcode vllm version or shouldn't we? @dtrawins @porlows1

@dkalinowski hardcoding vllm to v0.6.0 added